packages/story-editor/src/components/form/dateTime/datePicker.js

```diff
@@ -163,7 +163,7 @@ function DatePicker({ currentDate, onChange, onViewChange }) {
 DatePicker.propTypes = {
   onChange: PropTypes.func.isRequired,
   onViewChange: PropTypes.func,
-  currentDate: PropTypes.string,
+  currentDate: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
```

Comment:
Why is the currentDate sometimes a number? 🤔  Sounds like bad fixture data

Reply:
@swissspidy the reason is the `value` prop sent down from `DateTime` is actually number (confirmed this be logging typeof date) this is the case if the webstory is either in the following states [`draft`, `pending`, `auto-draft`] or if 
```
(date === modified || date === null);
```
which leads to passing of `displayDate` as value whose value type is number